### PR TITLE
WIP: Update tx interface to support select and get functions.

### PR DESCRIPTION
**Pull request by arijitAD**

### CODE REVIEW

**Comment by i-norden** [comment]
We don't need these methods, I had them previously but removed them.

**Comment by i-norden** [comment]
We don't need these on the Tx.

**Comment by i-norden** [comment]
We don't need these on the Tx. I previously had them and have since removed them.

**Comment by i-norden** [comment]
We don't need these on the Tx. I previously had them and have since removed them. Please don't work on-top of my ongoing line of work on this branch unless specified, we will end up duplicating efforts.

**Comment by i-norden** [comment]
I am mistaken since https://github.com/vulcanize/ipld-eth-server/pull/118 is already written to use the sql.Database abstraction from this repository, let's not dispose of those efforts.

File: statediff/indexer/database/sql/postgres/pgx.go [hunk 1]
```diff
@@ -211,6 +211,14 @@ type pgxTxWrapper struct {
 	tx pgx.Tx
 }
 
+func (t pgxTxWrapper) Get(ctx context.Context, dest interface{}, sql string, args ...interface{}) error {
+	panic("implement me")
```
Comment (i-norden) [inline statediff/indexer/database/sql/postgres/pgx.go]: Which means we need to add support for these new sql.Database methods to the pgx driver implementation.

Comment (arijitAD) [reply statediff/indexer/database/sql/postgres/pgx.go]: Yes, I have the code. I will update this in a while.

**Pull request closed without merging**